static/usr/lib/core/run-snapd-from-snap

```diff
@@ -9,6 +9,9 @@ set -eux
 run_on_unseeded() {
     SNAPD_BASE_DIR="/run/mnt/snapd"
 
+    [ -d /snap/snapd ] || mkdir -p /snap/snapd
+    ln -sf "${SNAPD_BASE_DIR}" /snap/snapd/current
```

Comment:
Why is this needed now and not in the past? Maybe this should have a comment.

Reply:
I have added a comment.